delayed_job.rb

```diff
@@ -1,12 +1,20 @@
-dep 'delayed job', :env, :user do
-  requires 'delayed_job.upstart'.with(env, user)
+dep 'delayed job', :env, :user, :queue do
+  requires 'delayed_job.upstart'.with(env, user, queue)
 end
 
-dep 'delayed_job.upstart', :env, :user do
+dep 'delayed_job.upstart', :env, :user, :queue do
   respawn 'yes'
+
+  queue.default 'global'
```

Comment:
Probably don't need a default value if we're checking for it later in `unless queue[/global/]`